t/02_request/14_uploads.t

```diff
@@ -182,5 +190,33 @@ for (qw/a b c/){
 }
 
 $resp =  dancer_response( 'POST', '/uploads', {files => $files});
-is length($resp->content), 3;
-like $resp->content, qr/a/;
+is $resp->content, 
+    join(';',"a:0cc175b9c0f1b6a831c399e269772661",
+    "b:92eb5ffee6ae2fec3ad71c777531578f",
+    "c:4a8a08f09d37b73795649038408b5f33"),
+    "Expected response for multi uploads";
+
+
+
+## test #23
+# create a file, 256MB of zeros
+$dest_file = File::Spec->catfile($dest_dir, "zeros");
+system("dd if=/dev/zero of=$dest_file count=250000 bs=1024");
```

Comment:
Righto - I'll wrap it in a check for an env var like that, so we can trigger it if we want to, but it not be used by default.

As nobody seems to have objected to the actual work I've done here, I plan to merge it to devel and get a trial release out this week (when I figure out releasing with dzil :) ) and watch for CPAN testers results - and also test it on our staging platform at work.